src/main/jhipster/src/main/webapp/app/shared/germplasm/service/germplasm-details.url.service.ts

```diff
@@ -25,4 +25,17 @@ export class GermplasmDetailsUrlService {
         return this.sanitizer.bypassSecurityTrustResourceUrl(GERMPLASM_DETAILS_URL + gid + queryParams);
 
     }
+
+    getUrlAsString(gid: any): string {
+
+        const queryParams = '?cropName=' + this.paramContext.cropName
```

Comment:
Can we extract this as constant:
`?cropName=' + this.paramContext.cropName            + '&amp;programUUID=' + this.paramContext.programUUID            + '&amp;authToken=' + this.paramContext.authToken            + '&amp;loggedInUserId=' + this.paramContext.loggedInUserId            + '&amp;selectedProjectId=' + this.paramContext.selectedProjectId;`

Reply:
done